ledger/sandbox-classic/src/test/lib/scala/platform/sandbox/LedgerResource.scala

```diff
@@ -93,6 +93,9 @@ private[sandbox] object LedgerResource {
           metrics = new Metrics(metrics),
           lfValueTranslationCache = LfValueTranslationCache.Cache.none,
           engine = new Engine(),
+          validatePartyAllocation = false,
+          enableAppendOnlySchema = true,
```

Comment:
This is changing the behavior, the test will be running append-only schema now. Was this intended?

Reply:
Changing all tests to use the append-only schema is something that we should do now, but I'd rather have this in a separate PR.

Reply:
Good point @rautenrieth-da. 
I thought it's intended to change to the append-only but I agree that it'd be better to do this in a separate PR.

Reply:
Oh, good point. Thanks!

Reply:
Reverted to `false`.